inductiva/molecules/simulators/ambertools.py

```diff
@@ -0,0 +1,52 @@
+"""Ambertools module of the API"""
+
+import pathlib
+from typing import Optional, List
+from uuid import UUID
+
+from inductiva import types
+from inductiva.simulation import Simulator
+
+
+class Ambertools(Simulator):
```

Comment:
I am still struggling with the fact that this is a set of tools (as the name explains) and we are still forcing this to fit the "Simulator" use case and then have "commands" being sent.

I suggest we discuss this with @hpenedones soon.